src/anonymous-credentials/schema.ts

```diff
@@ -982,6 +992,11 @@ export class CredentialSchema extends Versioned {
     return min >= 0 ? { type: this.POSITIVE_INT_TYPE } : { type: this.INT_TYPE, minimum: min };
   }
 
+  static parseDateType(node: object, parsingOpts: ISchemaParsingOpts, nodeName: string): object {
```

Comment:
`nodeName` is unused

Reply:
i recall having some problems with the boolean encoder without having these arguments there, but i removed it for this + boolean and seems fine now. maybe im misremembering or something changed since